### PR TITLE
MSPSDS-89 Use active storage to generate thumbnails for uploaded images

### PR DESCRIPTION
**Pull request by AnthonyPengelly**

### CODE REVIEW

**Comment by AnthonyPengelly** [comment]
Active storage lazily creates a thumbnail when it is first requested. It then stores this in S3 so next time it doesn't need to

File: app/views/investigations/show.html.erb [hunk 1]
```diff
@@ -22,7 +22,7 @@
   <% if @investigation.image.metadata["safe"].nil? %>
     <p>Still checking upload for viruses</p>
   <% elsif @investigation.image.metadata["safe"] %>
-    <%= image_tag(url_for(@investigation.image), alt: "Investigation") %>
+    <%= image_tag @investigation.image.variant(resize: "100x100"), alt: "Investigation" %>
```
Comment (broder) [inline app/views/investigations/show.html.erb]: So does this block the first time that it's requested?

Comment (AnthonyPengelly) [reply app/views/investigations/show.html.erb]: It only blocks the image loading. So when it runs this code, it doesn't do anything, it just puts in an image src in the html. When the browser tries to load the image, _then_ it builds the variant. (on my PC it only too 24ms to convert it)

Comment (broder) [reply app/views/investigations/show.html.erb]: That is really cool!

**Pull request merged**